### DISCUSSION TITLE
Is there a way to draw the following diagram?

### DISCUSSION THREAD

**Discussion by Tokubara** [terrastruct/d2#1409] [Q&A]
<img width="801" alt="Xnip2023-06-15_20-05-21" src="https://github.com/terrastruct/d2/assets/32409331/668790a4-60db-4268-919a-20ef4b08dc4d">

**Comment by alixander** [comment]
@Tokubara Something like this?

https://play.d2lang.com/?script=1JPPaoQwGMTveYphT8pisT0G2jco9NjDByWr8Q_NakgsdYl595LPuuv2DXrT-ZLMbybkt0WJIACjJj0XJzNWn6sAXJNLPN2EVlmJx5IFDi1xOPAfYz3sQSXK3SRVL8GIVjk9TAKIyXr81u7een9wFFGIflhHzCz3tAL4slY7iYVV3k9k-vNH6EDk0q1xSygRQdQ4VYUmm49dXjTZnMfQJYplI9kYNqsbHctn395ZJbO699aoS2rZh1ftvWo1iIhwwTNOOKKKvHjZ8uihGmtd_5tQbyY9Mz1Payybved_EvUDihdcc60f4icAAP__&

```
classes: {
  latex-block: {
    grid-rows: 2
    grid-gap: 10
    label: ""
    style.stroke-width: 0
    style.fill: transparent
  }
  lower-block: {
    label: ""
  }
}

in: {
  class: latex-block
  upper: |latex
    \\lim_{h \\rightarrow 0 } \\frac{f(x+h)-f(x)}{h}
  |
  lower: {
    class: lower-block
    msg: |latex
      \\displaylines{Message \\\\ y = b + c}
    |
  }
}

encoded: {
  class: latex-block
  upper: |latex
    \\lim_{h \\rightarrow 0 } \\frac{f(x+h)-f(x)}{h}
  |
  lower: {
    class: lower-block
    msg: |latex
      \\displaylines{Plaintext \\\\ p(X)}
    |
  }
}

in -> encoded: encode
```

![Screen Shot 2023-06-15 at 11 13 39 AM](https://github.com/terrastruct/d2/assets/3120367/8a6223f2-4075-4bd7-a7c2-7ee9aedae0ab)
